src/sentry/static/sentry/app/stores/configStore.tsx

```diff
@@ -46,7 +47,13 @@ const ConfigStore = Reflux.createStore({
         // e.g. unencoded "%"
       }
 
-      languageCode = queryString.lang || config.user.options.language || languageCode;
+      let queryStringLang = queryString.lang;
+
+      if (Array.isArray(queryStringLang)) {
```

Comment:
@lynnagara is this also fixing an implementation bug?

Reply:
It is, it could be an array if multiple values are provided in the URL